tests/dummy/config/environment.js

```diff
@@ -18,7 +18,8 @@ module.exports = function(environment) {
       // when it is created
     },
 
-    unstableApiURL: 'http://simonihmig.github.io/ember-bootstrap/'
+    unstableApiURL: 'http://simonihmig.github.io/ember-bootstrap/',
+    bootstrapVersion: process.env.BOOTSTRAPVERSION || 3
```

Comment:
I'd pull this logic down into a conditional, both to restrict the environment and to make changing the version simple for users.